docs/enhancements/1-appliedwork/README.md

```diff
@@ -0,0 +1,158 @@
+# New AppliedWork internal API
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/kubernetes-sigs/work-api)
+
+## Summary
+
+The proposed work would define a new AppliedWork internal API to represents a namespace scope workload that is applied on a managed cluster. 
+
+## Motivation
+
+The AppliedWork API will provide a definition on what is the workload that is applied on a managed cluster.
+The AppliedWork CR is to be an anchor on the managed cluster to ease garbage collection of applied workload.
+
+By leveraging this AppliedWork API:
+- Provide a view on the managed cluster the list of work has been applied.
+- Provide the source data for the work agent to delete resources on the managedcluster based on the appied work.
```

Comment:
it is also useful to clean up the applied resource when work agent is not running.